webpack.config.js

```diff
@@ -0,0 +1,69 @@
+var webpack = require('webpack');
+var path = require('path');
+var fs = require('fs');
+
+var externalModules = [
+	'mkfifo',
+	'sqlite3',
+	'mysql2',
+	'pg',
+	'mariasql',
+	'mssql',
+	'mysql',
+	'strong-oracle',
+	'oracle',
+	'oracledb',
+	'pg-query-stream'
+]
+
+// We iterate over the node modules and mark all optional dependencies as external
+var dirs = fs.readdirSync('./node_modules')
+for (let dir of dirs){
+	try {
+		let packageJson = JSON.parse(fs.readFileSync('./node_modules/' + dir + '/package.json'))
+		if (packageJson.optionalDependencies != null){
+			for (let key in packageJson.optionalDependencies) {
+				externalModules.push(new RegExp('^' + key + '(/.*)?$'))
```

Comment:
You might want to use `_.escapeRegExp(key)` just to make sure it doesn't have any special regex characters in it